tensorflow_probability/python/bijectors/masked_autoregressive.py

```diff
@@ -676,6 +679,50 @@ class AutoregressiveNetwork(tf.keras.layers.Layer):
   distribution.log_prob(np.ones((3, 2), dtype=np.float32))
   ```
 
+  The `conditional` argument can be used to instead build a conditional density.  
+  estimator. To do this the conditioning variable must be passed as a `kwarg`:
+
+  ```python
+  # Generate data as the mixture of two distributions.
+  n = 2000
+  c = np.r_[
+  np.zeros(n//2),
+  np.ones(n//2)
+  ]
+  x = np.r_[
+  np.random.randn(n//2).astype(dtype=np.float32) * 2,
+  np.random.randn(n//2).astype(dtype=np.float32) + 5
+  ]
+
+  # Density estimation with MADE.
+  made = tfb.AutoregressiveNetwork(params=2, hidden_units=[10, 10])
+
+  distribution = tfd.TransformedDistribution(
+    distribution=tfd.Sample(tfd.Normal(loc=0., scale=1.), sample_shape=[1]),
```

Comment:
I guess it's okay for a test, but MAF doesn't really do anything useful when it works on length=1 vectors. It'll just learn an unconditional shift + scale.

Reply:
Agreed. In this case, I'm just trying to make obvious how the feature works. Perhaps I could add a more complete example to a tutorial?

Reply:
Let's keep it as is, I was just thinking out loud.